pkg/pgmodel/migrations/generate.go

```diff
@@ -9,11 +9,19 @@
 package main
```

Comment:
I am not sure I like the approach of merging files in a directory at the generate level. I'd prefer to have execMigrationFiles go through the directory and execute the files one at a time. The reason is debuggability, If we merge files here it's very hard to determine which sql file an error originates from; if. we do it in execMigrationFiles then we can print out the filename along with the error. Also from the point of view of separation of concerns I think keeping the filesystem layer as simple as possible is a good goal.